app/assets/javascripts/controllers/auth_key_pair_cloud/auth_key_pair_cloud_controller.js

```diff
@@ -1,69 +0,0 @@
-ManageIQ.angular.app.controller('keyPairCloudFormController', ['$http', '$scope', 'keyPairFormId', 'miqService', function($http, $scope, keyPairFormId, miqService) {
-  var vm = this;
-
-  var init = function() {
-    vm.keyPairModel = {
-      name: '',
-      public_key: '',
-      ems_id: '',
-    };
-
-    vm.formId = keyPairFormId;
-    vm.afterGet = false;
-    vm.modelCopy = angular.copy( vm.keyPairModel );
-    vm.model = 'keyPairModel';
-    vm.ems_choices = [];
-    vm.saveable = miqService.saveable;
-    vm.newRecord = keyPairFormId === 'new';
-
-    ManageIQ.angular.scope = $scope;
-
-    miqService.sparkleOn();
-    $http.get('/auth_key_pair_cloud/ems_form_choices')
-      .then(getAuthKeyPairCloudFormDataComplete)
-      .catch(miqService.handleFailure);
-  };
-
-  var keyPairEditButtonClicked = function(buttonName, serializeFields) {
-    miqService.sparkleOn();
-
-    var url = '/auth_key_pair_cloud/create/' + keyPairFormId + '?button=' + buttonName;
```

Comment:
The `create` method in the `AuthKeyPairCloudController` can be deleted because of this and it can be removed from the `routes.rb`.

Reply:
Same goes for the referenced `create_finished` method...